velox/common/caching/SsdFile.cpp

```diff
@@ -491,6 +507,7 @@ void SsdFile::deleteFile() {
   }
   auto rc = unlink(filename_.c_str());
   if (rc < 0) {
+    ++stats_.errorsDeleteFile;
```

Comment:
deleteFile is not called in production so no need to track?

Reply:
Removed